server/lib/nl/detection/heuristic_detector.py

```diff
@@ -88,7 +87,8 @@ def detect(orig_query: str, cleaned_query: str,
         'err': str(e)
     })
   # Set the SVDetection.
-  sv_threshold_override = params.sv_threshold_override(dargs.mode)
+  sv_threshold_override = dargs.sv_threshold or params.sv_threshold_override(
+      dargs.mode)
```

Comment:
nit: how about passing `dargs` to sv_threshold_override?

Reply:
done